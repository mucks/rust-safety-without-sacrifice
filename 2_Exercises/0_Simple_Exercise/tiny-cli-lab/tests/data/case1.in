# Basics: numbers and text
ADD 2 3
MUL 4 5
UPPER hello world
REV rust is fun
COUNT banana
