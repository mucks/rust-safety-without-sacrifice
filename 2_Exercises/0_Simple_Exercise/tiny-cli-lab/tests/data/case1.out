5
20
HELLO WORLD
fun is rust
6
